about.html

```diff
@@ -155,16 +159,15 @@ <h2 class="event-date">2011</h2>
           </p>
         </div>
       </div>
-      <div class="container footer-right-reserved">
+
+      <div class="footer-right-reserved">
         <div class="footer-logo">
           <i class="far fa-closed-captioning"></i>
-          <div class="logo-word">
-            <p>creative commons korea</p>
-            <p class="footer-paragraph-right-reserved">
-              @2023 Creative Commons Korea Some Rights Reserved
-            </p>
-          </div>
+          <p>creative commons korea</p>
         </div>
+        <p class="footer-paragraph-right-reserved">
+          @2023 Creative Commons Korea Some Rights Reserved
+        </p>
       </div>
     </footer>
```

Comment:
- [ ] Good job working on the footer and ensuring the width matches the design, but as pointed out by the previous reviewer please ensure that the background color matches the design both on desktop and mobile.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/210765512-2b291e50-fb49-4ffd-9b2e-cb52fa36fda6.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/210765588-ccfa2ca9-f210-480c-aa38-7ae4a2491a50.png)